build.sbt

```diff
@@ -43,14 +43,14 @@ libraryDependencies ++= Seq(
     "com.gu" %% "membership-common" % "0.159",
     "com.gu" %% "memsub-common-play-auth" % "0.2",
     "com.gu" %% "content-authorisation-common" % "0.1",
-    "com.github.nscala-time" %% "nscala-time" % "2.0.0",
+    "com.github.nscala-time" %% "nscala-time" % "2.10.0",
     "net.kencochrane.raven" % "raven-logback" % "6.0.0",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
     "org.scalatest" %% "scalatest" % scalatestVersion % "test",
     "org.scalactic" %% "scalactic" % scalatestVersion % "test",
     "org.seleniumhq.selenium" % "selenium-java" % "2.48.2" % "test",
-    "com.gocardless" % "gocardless-pro" % "1.8.0",
-    "com.squareup.okhttp" % "okhttp" % "2.4.0",
+    "com.gocardless" % "gocardless-pro" % "1.16.0",
```

Comment:
This appears to have caused an outage on our site, where https://subscribe.theguardian.com/checkout/check-account would hang indefinitely, fixed with https://github.com/guardian/subscriptions-frontend/pull/363